lib/phoenix_html/form.ex

```diff
@@ -284,7 +284,7 @@ defmodule Phoenix.HTML.Form do
        form: form,
        id: input_id(form, field_as_string),
        name: input_name(form, field_as_string),
-       value: input_value(form, field_as_string)
+       value: input_value(form, field)
```

Comment:
This change may not be strictly necessary– it raised when using map access (`f[:name]`) with `<.form :let={f} for={@changeset}>` in LiveView.